kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -190,6 +191,9 @@ public class S3SinkConnectorConfig extends StorageSinkConnectorConfig {
   public static final String ELASTIC_BUFFER_INIT_CAPACITY = "s3.elastic.buffer.init.capacity";
   public static final int ELASTIC_BUFFER_INIT_CAPACITY_DEFAULT = 128 * 1024;  // 128KB
 
+  public static final String TOMBSTONE_ENCODED_PARTITION = "tombstone.encoded.partition";
+  public static final String TOMBSTONE_ENCODED_PARTITION_DEFAULT = "tombstone";
```

Comment:
is it a partition or a folder name?

Reply:
Made the documentation of this config more clear.